ray-operator/config/samples/pytorch-text-classifier/fine-tune-pytorch-text-classifier.py

```diff
@@ -131,15 +131,16 @@ def train_func(config):
     # The checkpoints and metrics are reported by `RayTrainReportCallback`
     run_config = RunConfig(
         name="ptl-sent-classification",
+        storage_path="/mnt/cluster_storage",
```

Comment:
Why do we need this change? This change may cause users fail to follow [this doc](https://docs.ray.io/en/latest/cluster/kubernetes/examples/rayjob-kueue-priority-scheduling.html#kuberay-kueue-priority-scheduling-example).

Reply:
sorry, this was leftover from the last change, reverting it